epix/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "epix",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://epix.zone/",
+  "pretty_name": "Epix",
+  "chain_type": "cosmos",
+  "chain_id": "epix_4242-0",
+  "bech32_prefix": "epix",
+  "daemon_name": "epixd",
+  "node_home": "$HOME/.epixd",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "aepix",
+        "fixed_min_gas_price": 1000000000000,
+        "low_gas_price": 1000000000000,
+        "average_gas_price": 2000000000000,
+        "high_gas_price": 3000000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "aepix"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/EpixZone/Epix",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": [
+      "v1.0.0"
+    ],
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/EpixZone/Epix/refs/heads/main/Networks/Mainnet/epix_4242-0/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+          "v1.0.0"
+        ],
+        "next_version_name": "",
+        "sdk": {
+          "type": "cosmos_sdk",
+          "version": "0.46"
+        },
+        "consensus": {
+          "type": "tendermint",
+          "version": "0.34"
+        },
+        "ibc": {
+          "version": "v6.1.0"
```

Comment:
"type" is also required. Most common is "go"

Reply:
Fixed, thank you.